pages/commande/index.js

```diff
@@ -38,6 +99,46 @@ export default function Commande() {
     setActiveLivraison(!activeLivraison);
     setActivePayment(!activePayment);
     setCheckedLivraison(!checkedLivraison);
+    axios
+      .post("/api/order", {
+        deliveryPhoneNumber: phonePartner,
+        deliveryFirstName: partnerFirstName,
+        deliveryLastName: partnerLastName,
+        deliveryStreet: partnerAddress,
+        deliveryZip: partnerZip,
+        deliveryCity: partnerCity,
+        deliveryArrivalTime: userHourArrived,
+        comment: userCommentary,
+        orderNumber: "123456789",
+        startDate: "",
```

Comment:
dates en dur
on devrait générer un numéro de commande + orderDate sur le back plutôt

Reply:
Pour le OrderNumber, il es noté en string sur le schéma prisma, est-ce que je peux le modifier pour qu'il s'auto-incrémente plutôt ? sinon comment faire pour en créer un ?

Reply:
Ben on a pas de contrainte particulière pour ces numéro de commande, si ? Pourquoi pas utiliser une chaine aléatoire ? Si on peut utiliser de l'aléatoire je vous laisse regarder comment on peut faire ça en JS c'est pas bien compliqué :)